interfaces/builtin/desktop.go

```diff
@@ -284,12 +287,34 @@ type desktopInterface struct {
 	commonInterface
 }
 
-func (iface *desktopInterface) fontconfigDirs() []string {
+func (iface *desktopInterface) shouldMountFontCache(attribs interfaces.Attrer) (bool, error) {
+	value, ok := attribs.Lookup("mount-font-cache")
```

Comment:
I'll be the first to start 2020 name bike shedding contest. Wondering about the name a bit. Maybe it should be called `private-font-cache` so that it's more about the actual intent of having a private cache rather than the actual action of mounting it which is an implementation detail after all.

Reply:
My main concerns with the `private-font-cache` name is that (a) you could achieve a private cache without this setting with an appropriate fontconfig configuration, and (b) people might think it implies snapd will manage that cache some how (it's not going to do anything with the extra fc-cache binaries shiped with snapd, for instance).

I picked this name because it more closely reflects what it actually does.  With that said, I'm open to other suggestions though.

Reply:
I think for claritiy this should be either be called:

* host-font-cache
* or more verbose  mount-host-font-cache

Reply:
I've renamed the attribute to `mount-host-font-cache`.